openapi/rhose-api.yaml

```diff
@@ -592,7 +592,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/JsonNode'
+                type: object
```

Comment:
@r00ta Is there a way to keep this `object` declaration for the schema type without having also
```
additionalProperties:
            type: object
```
for the `parameters` definitions (lines 739 and 1038)?

The `additionalProperties` is generating a `{ [key:string]: object}` for parameters in the sdk. It's better to have them declared just as `object`, like before.

Reply:
@jomarko yes, the parameters are fine as they are. I was trying to understand if we can keep also the `type: object` on the schema definition.

Reply:
I do apologize, I forgot this PR included also this fix. I've reopened it :+1: